vue/test/components/steps/steps.test.js

```diff
@@ -0,0 +1,75 @@
+const assert = require("assert");
+const base = require("../../base");
+
+describe("Steps", () => {
+    it("should initialize the steps", () => {
+        const component = base.getComponent("Steps", {
+            props: { steps: ["A", "B", "C"], step: 1 }
+        });
+        assert.strictEqual(component.exists(), true);
+    });
+    it("should only highlight the current step", () => {
+        const component = base.getComponent("Steps", {
+            props: { steps: ["A", "B", "C", "D", "E"], step: 2 }
+        });
+        const steps = component.findAll(".step .highlight");
+        assert.strictEqual(steps.at(0).isVisible(), false);
+        assert.strictEqual(steps.at(1).isVisible(), true);
+        assert.strictEqual(steps.at(2).isVisible(), false);
+        assert.strictEqual(steps.at(2).isVisible(), false);
+        assert.strictEqual(steps.at(2).isVisible(), false);
+    });
+    it("should change to clicked step and emit change", async () => {

```

Comment:
Check that the highlight changed.

Reply:
Asserting internal data is like documenting internal methods: even if it's great when you do it and it helps a lot on future developments, it is neither absolutely necessary nor enough.

What I mean is that unit tests usually treat components under test as black boxes: you don't care how they're implemented, you just care of what you can see outside their interface (UI, method calls, events, side-effects like writing to file, etc).